### ISSUE TITLE
[BUG] "Provider Company Name" required error shows up when "Issue Type" is NOT Scooter/Electric Bike issues (micro-modal)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#6414]
> What application are you using?

Data Tracker

> Describe the problem.

When I try to make a new CSR  it will not let me because its saying it requires a provider/company and that is a detail meant for scooters 

> Website Address

Parking Enterprise Portal

Internet Browser: Chrome

> Requested By
Jasmine L.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/60ca237951ffc0001ea07797/screenshot20210616110531.png) (136.14kb)

Request ID: DTS21-102195

**Comment by dianamartin** [comment]
I was troubleshooting and couldn't figure out why the "Provider Company Name" field kept showing up. There is a display rule that is set to show it when the "Issue Type" is NOT Scooter/Electric Bike issues (micro-modal). I tried a few things
- modified the Display Rules 
- Set default to new choice "NO PROVIDER" on field: "Provider Company Name"
- removed the "Required" on "Provider Company Name"
  - this was the only thing that worked
 
Messaged Jasmine and told her what I did and that I created a test record that she could use and just to edit it. 
- Will put in Knack ticket, bc the behavior seems like a bug
